src/Compilers/CSharp/Test/Symbol/Symbols/MissingSpecialMember.cs

```diff
@@ -1679,6 +1679,7 @@ static void Main()
 }";
             var compilation = CreateCompilationWithMscorlib461(source);
             compilation.MakeMemberMissing(SpecialMember.System_String__ConcatStringStringStringString);
+            compilation.MakeMemberMissing(SpecialMember.System_String__Concat_4ReadOnlySpans);
```

Comment:
>compilation.MakeMemberMissing(SpecialMember.System_String__Concat_4ReadOnlySpans); [](http://example.com/codeflow?start=12&length=82)

Could you please elaborate why changes to this unit-test are necessary (including the change to the ```source```)?

Reply:
Sure. We needed to change the source because it used 5 strings, not 4. Previously, since we built from the bottom up, we'd hit the 4-string level and break. Now, however, when we see 5 strings we go straight for `string.Concat(params string[])`, and never check for the 4-string version. I believe I can remove this MakeMemberMissing call though, it's an artifact of an earlier iteration when I changed the compilation target instead, and I forgot to revert this line.